src/coreclr/src/System.Private.CoreLib/src/System/Threading/ThreadPool.CoreCLR.cs

```diff
@@ -260,7 +260,7 @@ public static void GetAvailableThreads(out int workerThreads, out int completion
             get;
         }
 
-        private static RegisteredWaitHandle RegisterWaitForSingleObject(  // throws RegisterWaitException
+        private static RegisteredWaitHandle RegisterWaitForSingleObject(
```

Comment:
Did your auto-fixer remove this comment, or did you do so manually?

Reply:
I removed it manually by addressing review comment from my old PR https://github.com/dotnet/runtime/pull/35147#discussion_r411087780